.github/workflows/gemini-invoke.yml

```diff
@@ -107,17 +52,18 @@ jobs:
           git config --global user.name 'gemini-cli[bot]'
           git config --global user.email 'gemini-cli[bot]@users.noreply.github.com'
 
-      - name: 'Run Gemini'
+      - name: 'Run Gemini CLI'
         id: 'run_gemini'
-        uses: './'
+        uses: 'google-github-actions/run-gemini-cli@main' # ratchet:exclude
```

Comment:
this is good for consistency with examples too, but i think it makes it hard to test the changes in `action.yml` within a PR before merging the change - but i suppose one can temporarily use `./` in a branch when testing with workflow dispatch? 🤔

Reply:
makes sense, will add instructions in our contributing guide about testing in private repos 🙏🏾

Reply:
cc @leehagoodjames